NewForce-at-Mountwest/bangazon-workforce-management-the-lemony-slaps
# Ticket Number 4 - HR Should Be able to Edit an Employee

Description:
Given user is viewing an employee
When user clicks on the Edit link
Then user should be able to edit the last name of the employee
Or change the department to which the employee is assigned
Or change the computer assigned to the employee

If you want to see the ticket for yourself then click the link or copy and paste the link below:
https://trello.com/c/rBTGL7Z0/4-4-hr-should-be-able-to-edit-an-employee

Steps to test:

Go ahead and pull down these changes to your branch.
Once you've done that, go ahead and execute and run the app.
The next thing to do is to change the url by adding /employees to the end, then click on the Edit Link to the Right of the Employee that you would like to edit.
The form should load with the Employees current information displayed. You can then change any of the information displayed, then Select the Save Button or to Not Save the Edits, Click the Back To List Link.
You should then be taken to a list of all of the employees in the Bangazon work force.

Cheers.
